app/src/main/res/layout/tool_details_page_description.xml

```diff
@@ -14,11 +15,10 @@
         style="@style/Widget.GodTools.Tools.Details.Page.Content"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:autoLink="web"
         android:paddingLeft="@dimen/tool_details_margin_horizontal"
         android:paddingTop="@dimen/tool_details_page_padding_vertical"
         android:paddingRight="@dimen/tool_details_margin_horizontal"
         android:paddingBottom="@dimen/tool_details_page_padding_vertical"
-        android:text="@{ModelUtils.getTranslationDescription(translation, tool, context)}"
+        app:webLinkText="@{ModelUtils.getTranslationDescription(translation, tool, context).toString()}"
```

Comment:
using `toString()` here will break fonts for Sinhala and Tibetan on older devices